### ISSUE TITLE
Question about homework3

### ISSUE THREAD

**Issue by jaeyeonkim99** [swsnu/swppfall2021#82]
Before submitting, I want to make sure about some points.

1. Does different tabs only means that they are displayed differently on the url? Do not need other addiotional features for tab? 
2. Is it right that there isn't any requirements for the "id" of the buttons for article titles on article list pages?

Thank you

**Comment by kdh0102** [comment]
> Does different tabs only means that they are displayed differently on the url? Do not need other addiotional features for tab?

Yes. It is OK not to implement anything not specified in the spec.

> Is it right that there isn't any requirements for the "id" of the buttons for article titles on article list pages?
 
Yes, no id required for each article title.
